src/PresentationalComponents/TableToolbar/TableToolbar.js

```diff
@@ -1,25 +1,42 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import propTypes from 'prop-types';
 
 import classNames from 'classnames';
 
 import './TableToolbar.scss';
 
-const TableToolbar = ({ className, children, ...props }) => {
+const TableToolbar = ({ results, className, children, ...props }) => {
 
     const tableToolbarClasses = classNames(
         'ins-c-table__toolbar',
         className
     );
 
+    function generateResults(results) {
```

Comment:
I might have been too fast with the approve button. Can we move this function above `TableToolbar` component? So it is not recreated everytime we re render table toolbar?